editor/js/ui/gitConfig.js

```diff
@@ -0,0 +1,312 @@
+                    },body);
+                });
+            var updateForm = function() {
+                // sshkeysList.editableList('empty');
+                // if (activeProject.git.hasOwnProperty('remotes')) {
+                //     for (var name in activeProject.git.remotes) {
+                //         if (activeProject.git.remotes.hasOwnProperty(name)) {
+                //             remotesList.editableList('addItem',{name:name,urls:activeProject.git.remotes[name]});
+                //         }
+                //     }
+                // }
+                // var sshkeyFiles = ["/User/hideki/.node-red/sshkeys/node-red-ssh-test01", "/User/hideki/.node-red/sshkeys/node-red-ssh-test02"];
+                // if ( sshkeyFiles ) {
+                //     sshkeyFiles.map(function(sshkeyFilePath) {
+                //         sshkeysList.editableList('addItem', {path: sshkeyFilePath, data: 'XXXXXXX'});
+                //     });
+                // }
+            }
+
+            sendRequest({
+                url: "settings/user",
+                type: "GET",
+                responses: {
+                    0: function(error) {
+                        console.log(error);
+                    },
+                    200: function(result) {
+                        console.log(result);
+                        if ( result ) {
+                            gitUsernameLabel.text(result.name);
+                            gitUsernameInput.val(result.name);
+                            gitEmailLabel.text(result.email);
+                            gitEmailInput.val(result.email);
+                        }
+                    },
+                    400: {
+                        'unexpected_error': function(error) {
+                            console.log(error);
+                        }
+                    }
+                }
+            })
+
+            updateForm();
+        }
+
+        function createSettingsPane(activeProject) {
+            var pane = $('<div id="user-settings-tab-gitconfig" class="project-settings-tab-pane node-help"></div>');
+            createRemoteRepositorySection(pane);
+            return pane;
+        }
+    
+        var popover;
+    
+        var utils;
+        function init(_utils) {
+            utils = _utils;
+            RED.userSettings.add({
+                id:'gitconfig',
+                title: "Git config", // TODO: nls
+                get: createSettingsPane,
+                close: function() {
+                    if (popover) {
+                        popover.close();
+                        popover = null;
+                    }
+                }
+            });
+    
+        }
+
+        function addSpinnerOverlay(container) {
+            var spinner = $('<div class="projects-dialog-spinner"><img src="red/images/spin.svg"/></div>').appendTo(container);
+            return spinner;
+        }
+            
+        function sendRequest(options,body) {
```

Comment:
We should not duplicate this code here. As I mention in other comment, by changing how this module is initialised, you can pass in a reference to the `projectsAPI` object which gives you access to the `sendRequest` function defined in `projects.js`.